docs/modules/ROOT/pages/how-tos/use-exoscale.adoc

```diff
@@ -0,0 +1,32 @@
+= Use Exoscale
+
+The following steps show how to set up Terraform with Exoscale.
+
+. Set up 2 new API keys in https://portal.exoscale.ch[portal.exoscale.ch].
+  One is for read-only access, the other for mutating operations.
+. Configure component parameters.
++
+[source,yaml]
+----
+openshift4_terraform:
+  provider: exoscale
+  terraform_variables:
+    worker_count: 3
+    rhcos_template: TheTemplateNameForRHCOS
+    lb_exoscale_api_key: KeyForLoadBalancersToManageEIP
+    lb_exoscale_api_secret: Secret
+    worker_size: Extra-large
+    base_domain: ${openshift:baseDomain}
+    ignition_ca: |-
+      -----BEGIN CERTIFICATE-----
+      ...
```

Comment:
not sure if those are the minimum required parameters or if the example could be generally better